### DISCUSSION TITLE
Plate-based (Smart-seq) scRNA-seq

### DISCUSSION THREAD

**Discussion by Rads2512** [alexdobin/STAR#1686] [General]
Hi Alex! I've been trying to use STARsolo on raw fastq files with no barcodes. There isn't much documentation on using these files and I have been running into the following error "EXITING because of fatal input ERROR: could not open readFilesManifest=Read1_file_name" when I try to run the first line of code. I have been trying to run this only on 1 fastq file with paired-end reads. 
I made a tab-delimited file with the /path/to/read1, /path/to/read2 and cell_id according to (Read1-file-name \t Read2-file-name \t Cell-id) documentation.
Could you help me out somehow with how to debug this?

**Comment by alexdobin** [comment]
Hi Radhika,

this error means that STAR cannot open the manifest file, i.e. some problems with path or permissions.
